parser/Builder.cc

```diff
@@ -1085,7 +1085,7 @@ class Builder::Impl {
         core::NameRef nm;
 
         if (name != nullptr) {
-            loc = loc.join(tokLoc(name));
+            loc = tokLoc(name);
```

Comment:
This is a minor thing, but instead of overwriting `loc` in just this case, do you mind changing it so that `loc` is not given an initial value, and instead it's written to once in each of the `if` branches?

Reply:
i went ahead and did this. it'll merge when the tests pass.